app/uploaders/evss_claim_document_uploader.rb

```diff
@@ -13,7 +13,7 @@ def initialize(user_uuid, tracked_item_id)
   end
 
   def store_dir
-    store_dir = "disability_claim_documents/#{@user_uuid}"
+    store_dir = "evss_claim_documents/#{@user_uuid}"
```

Comment:
This could cause an issue with any queued jobs that have documents uploaded to the old `store_dir` and then try to access them from the new `store_dir`